src/controllers/update/overseas.entity.update.details.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { NextFunction, Request, Response } from "express";
```

Comment:
similar comment to the one above, the existing register controller for the same page is EntityController so maybe this one could be UpdateEntityController?